tools/cellprofiler/measure_texture.xml

```diff
@@ -0,0 +1,135 @@
+import json
+import sys
+import os
+
+FOURSPACES=@SPACES@
+NEWLINE=@NEWLINE@
+
+input_json_path = sys.argv[1]
+input_pipeline= sys.argv[2]
+
+params = json.load(open(input_json_path, "r"))
+
+def writemi():
+  f.write(NEWLINE)
+
+  f.write("MeasureTexture:[module_num:"+str(new_count)+"|svn_version:\\'Unknown\\'|variable_revision_number:5|show_window:False|notes:\\x5B\\'PARAMS\\x3A\\', \\'- Texture scale\'\x5D|batch_state:array(\\x5B\\x5D, dtype=uint8)|enabled:True|wants_pause:False]"+NEWLINE)  
+
+  img_count = str(len(params['rpt_image_to_measure']))
+  obj_count = str(len(params['con_img_obj']['rpt_obj_to_measure']))
+  scale_count = str(len(params['rpt_scale_to_measure']))
+
+  img_obj = params['con_img_obj']['img_obj']
+
+  f.write(FOURSPACES + "Hidden:" + img_count + NEWLINE)
+  f.write(FOURSPACES + "Hidden:" + obj_count + NEWLINE)
+  f.write(FOURSPACES + "Hidden:" + scale_count + NEWLINE)
+
+  for img in params['rpt_image_to_measure']:
+    f.write(FOURSPACES + "Select an image to measure:" + img['image_to_measure']+ NEWLINE)
+
+  if img_obj =="Images"  :
+    f.write(FOURSPACES + "Select objects to measure:None"+ NEWLINE)
+  else:
+    for obj in params['con_img_obj']['rpt_obj_to_measure']:
+      f.write(FOURSPACES + "Select objects to measure:" + obj['obj_to_measure']+ NEWLINE)
+
+  for scale in params['rpt_scale_to_measure']:
+    f.write(FOURSPACES+ "Texture scale to measure:" + str(scale['scale_to_measure'])+ NEWLINE)
+
+  f.write(FOURSPACES + "Measure images or objects?:" + params['con_img_obj']['img_obj']+ NEWLINE)
+
+
+with open(input_pipeline) as fin:
+  lines = fin.readlines()
+  
+  k,v= lines[4].strip().split(':')
+
+  module_count = int(v)
+  new_count = module_count+1
+  lines[4]= k+":"+str(new_count)+"\n"
+  
+  with open("output","w") as f:
+    f.writelines(lines)
+    writemi()
+
+f.close() 
+        </configfile>    
+    </configfiles>
+  
+    <inputs>
+
+        <expand macro="input_pipeline_macro" />
+        <repeat name="rpt_image_to_measure" title="Add another image">
+            <param name="image_to_measure" label="Select an image to measure" type="text" >
+                <expand macro="text_validator" />
+            </param>
+        </repeat>
+
+        <conditional name="con_img_obj">
+            <param name="img_obj" label="Measure images or objects?" type="select">
+                <option value="Images">Images</option>
+                <option value="Objects">Objects</option>
+                <option value="Both">Both</option>
+            </param>
+            <when value="Objects">
+                <expand macro="measureImgObj" />
+            </when>
+            <when value="Both">
+                <expand macro="measureImgObj" />
+            </when>
+            <when value="Images" />
+        </conditional>
+        <repeat name="rpt_scale_to_measure" title="Add another scale">
+            <param name="scale_to_measure" label="Texture scale to measure" type="integer" value="3" min="0.0" />

```

Comment:
it's this, left?

```suggestion
            <param name="scale_to_measure" label="Texture scale to measure" type="integer" value="3" min="0" />

```

Reply:
Yes, but there are more :)

Reply:
I remember I fix this but I will check again.

I think there's another issue, for example the common.xml does not have any integer or float. Planemo test common.xml with --galaxy_root parameter works but planemo test without it does not work..

Reply:
maybe you are testing a different version locally?